src/main/java/seedu/address/model/person/Person.java

```diff
@@ -140,4 +140,16 @@ public String toString() {
         return getAsText();
     }
 
+    @Override
+    public int compareTo(Person o){
+        if(this.getName().toString().compareToIgnoreCase(o.getName().toString()) > 0){
```

Comment:
Why not just directly return the result of this comparison? They will still be negative, 0 or positive and comply with the requirements of `compareTo()`.

Reply:
Im on it, Thx for the suggestion!